### PR TITLE
Create notification

### PR DESCRIPTION
**Pull request by VuManhHung97**
Register for channel notification
Create the properties of the notification
Handle when click on notification , open application

https://edu-redmine.sun-asterisk.vn/issues/12518

notification

### CODE REVIEW

**Comment by VuManhHung97** [comment]
ready

File: app/src/main/java/com/sun_asterisk/music_65/screen/notification/SongNotification.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            .addAction(mAction)
+            .addAction(R.drawable.ic_next, mContext.getString(R.string.next),
+                pendingIntentAction(CommonUtils.Action.ACTION_NEXT))
+            .setShowWhen(false)
+            .setPriority(NotificationCompat.PRIORITY_MAX)
+            .setVisibility(VISIBILITY_PUBLIC)
+            .setContentIntent(pendingIntentActivity())
+            .setStyle(
+                new android.support.v4.media.app.NotificationCompat.MediaStyle().setShowActionsInCompactView(
+                    0, 1, 2));
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/music_65/screen/notification/SongNotification.java]: hard code. đặt tên thành các action button đi e

File: app/src/main/java/com/sun_asterisk/music_65/screen/notification/SongNotification.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+            String description = mContext.getString(R.string.descriptionChannel);
+            int important = NotificationManager.IMPORTANCE_LOW;
+            mNotificationChannel = new NotificationChannel(NOTIFICATION_CHANNEL, name, important);
+            mNotificationChannel.setDescription(description);
+            mNotificationChannel.enableLights(true);
+            mNotificationChannel.setLightColor(Color.GREEN);
+            mNotificationManager.createNotificationChannel(mNotificationChannel);
+        }
+        mAction =
+            new NotificationCompat.Action(R.drawable.ic_pause, mContext.getString(R.string.pause),
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/music_65/screen/notification/SongNotification.java:48]: check notification <8.0 đi e

**Comment by VuManhHung97** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**